STATUS.md

```diff
@@ -4,8 +4,8 @@ Test Case                                                 | Frames      |     |
 :-------------------------------------------------------- | :---------: | --- | :---------------------------
 [`lc-rta-1-08-733`](https://youtu.be/HPcvNS8QFVI)         | 4531 / 4531 | ✔️ |
 [`mmm-rta-1-15-273`](https://youtu.be/ozaXzEXFeHM)        | 4923 / 4923 | ✔️ |
-[`mg-rta-0-19-758`](https://youtu.be/ui01yrKCwa0)         | 628 / 1596  | ❌ | Mushroom
-[`mg-ng-rta-1-40-273`](https://youtu.be/8-0Xetey5xY)      | 877 / 6422  | ❌ | Mushroom
+[`mg-rta-0-19-758`](https://youtu.be/ui01yrKCwa0)         | 1596 / 1596  | ✔️ |
+[`mg-ng-rta-1-40-273`](https://youtu.be/8-0Xetey5xY)      | 6422 / 6422  | ✔️ |
```

Comment:
Since numbers went from 3 digits to 4 digits, delete whitespace character so that pipes are aligned. Sorry for being this pedantic
```suggestion
[`mg-rta-0-19-758`](https://youtu.be/ui01yrKCwa0)         | 1596 / 1596 | ✔️ |
[`mg-ng-rta-1-40-273`](https://youtu.be/8-0Xetey5xY)      | 6422 / 6422 | ✔️ |
```